### PR TITLE
add splash screen

### PR DESCRIPTION
**Pull request by im-chang**
add splash screen for app

### CODE REVIEW

File: js/App.js [hunk 1]
```diff
@@ -1,38 +1,64 @@
...
+  }
+
+  hide_splash = () => {
+    this.setState({
+      isVisible: false,
+    });
+  };
+
+  componentDidMount() {
+    var that = this;
```
Comment (10thfloor) [inline js/App.js]: This is an anti-pattern. Please use an arrow function in your setTimeout and avoid using the var keyword.

**Review by 10thfloor — CHANGES_REQUESTED** [review]

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
https://medium.com/handlebar-labs/how-to-add-a-splash-screen-to-a-react-native-app-ios-and-android-30a3cec835ae

Use this for the splash screen and remove all this code related to the splash screen. Thank you

File: js/App.js [hunk 2]
```diff
@@ -1,38 +1,49 @@
...
+      isVisible: false,
+    });
+  };
+
+  componentDidMount() {
+    let hide = this.hide_splash;
+
+    setTimeout(() => {
+      hide();
+    }, 1000);
```
Comment (siddharthparmar7) [inline js/App.js:25]: You should not be hiding your splash screen manually after a second.

File: js/App.js [hunk 3]
```diff
@@ -1,38 +1,49 @@
...
-    'Shake or press menu button for dev menu',
-});
+import styles from './styles';
+import { View, StatusBar, Text, Image, Alert } from 'react-native';
 
 export default class App extends Component {
+  constructor() {
+    super();
+
+    this.state = {
```
Comment (siddharthparmar7) [inline js/App.js:9]: You do not need state here. Please follow the article we gave you to add a splash screen.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**